src/components/Button.jsx

```diff
@@ -1,63 +1,178 @@
-a, button{
-  text-decoration: none;
-  outline:none;
-  color: #001426;
-  cursor: pointer;
-}
-.primary{
-  background-color: red;
-  border: 2px solid black;
-  width: 6.5rem;
-  height: 2.4rem;
-}
-.secondary{
-  background-color: green;
-  width: 8rem;
-}
-.textBtn{
+  /* sample styling, will get overwritten to match figma template */
+  a,
+  button {
+    text-decoration: none;
+    outline: none;
+    color: var(--blizzard-black);
+    cursor: pointer;
+    svg {
+      margin-right: -0.25rem;
+      padding: 0.12rem;
+    }
+  }
+  .primary {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 0.5rem 1rem;
+    width: 7rem;
+    height: 2.3rem;
+    background: var(--white);
+    border: 3px solid var(--dark-red);
+    box-shadow: 2px 2px 4px rgba(0, 0, 0, 0.25);
+    border-radius: 2.2px;
+    &.small {
+      width: 6rem;
+      height: 2rem;
+    }
+    svg {
+      fill: red;
+    }
+  }
+  .primary,
+  .primary.small {
+    &:hover {
+      color: var(--white);
+      background: var(--dark-red);
+      box-shadow: 2px 3px 4px var(--shadow-black);
+      border-radius: 2.2px;
+      svg {
+        fill: var(--white);
+      }
+    }
+    &:active {
+      background: var(--darker-red);
+      box-shadow: none;
+    }
+    &:focus {
+      border: 2px solid var(--white);
+      outline: 2px solid var(--medium-blue);
+      border-radius: 2.2px;
+      background: var(--dark-red);
+      color: var(--white);
+      svg {
+        fill: var(--white);
+      }
+    }
+    &:disabled {
+      color: var(--dark-grey);
+      background: var(--light-grey);
+      border: none;
+      box-shadow: none;
+      svg {
+        fill: var(--dark-grey);
+      }
+    }
+  }
+  .secondary {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 0.5rem 1rem;
+    width: 5rem;
+    height: 2.31rem;
+    background: var(--white);
+    border: 2px solid var(--blizzard-blue);
+    box-shadow: 1px 2px 4px rgba(0, 0, 0, 0.25);
+    border-radius: 2.2px;
+    &.small {
+      width: 4.63rem;
+      height: 2.13rem;
+    }
+  }
+  .secondary,
+  .secondary.small {
+    &:hover {
+      box-shadow: 1px 2px 6px rgba(0, 0, 0, 0.6);
+    }
+    &:active {
+      background: var(--blizzard-blue);
+      box-shadow: none;
+    }
+    &:focus {
+      background: var(--white);
+      border: 1.5px solid var(--medium-blue);
+      border-radius: 3.3px;
+      outline: 2px solid var(--medium-blue);
+    }
+    &:disabled {
+      border: 2px solid var(--light-grey);
+      color: var(--dark-grey);
+    }
+  }
 
-}
-.small{
-  width: 4rem;
-  height: 2rem;
-}
-`;
+  .textBtn {
+    width: 8.5rem;
+    height: 1.31rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 0px;
+    border: none;
+    color: var(--dark-red);
+    border-radius: 2.2px;
+    background: none;
+    &.small {
+      height: 1.12rem;
+      width: 5.75rem;
+      gap: 8px;
+    }
```

Comment:
Let's add some hover effects. It looks like the arrow will animate to the right a bit. We'll want to use the "transition" rule in CSS to make that happen.

Reply:
I have a few questions about the transition rule. I've completed the other requested changes

Reply:
Gotcha. Yeah basically you call [transition](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Transitions/Using_CSS_transitions) as a regular CSS declaration. And then for your :hover pseudo class you set a different rule (for example left padding in this case). And CSS will know that you want to animate that change not just jump to it. 

CSS also has a more involved "animate" rule, but I've found that 90% of the time `transition` works perfectly. 

So to pseudo code this, you'll want to set the transition on the padding of the txtBtn class. Then on hover you'll set the padding a different amount than the not-hover padding and CSS will do the rest.